torchvision/prototype/transforms/_augment.py

```diff
@@ -10,6 +10,8 @@
 from torchvision.prototype import features
 from torchvision.prototype.transforms import functional as F, InterpolationMode
 
+from ..features._image import ImageType, TensorImageType
```

Comment:
I would avoid relative imports that span more than one "major" namespace, i.e. relative imports within `torchvision.prototype.transforms` are ok, but importing something from `torchvision.prototype.features` should be done by an absolute import:

```suggestion
from torchvision.prototype.features._image import ImageType, TensorImageType
```

My argument is somewhat weakened by the fact that all of the imports are internally any, so no strong opinion here.

Reply:
Makes sense, done.